redis/commands/core.py

```diff
@@ -1926,6 +1926,19 @@ def brpoplpush(self, src, dst, timeout=0):
             timeout = 0
         return self.execute_command("BRPOPLPUSH", src, dst, timeout)
 
+    def lmpop(self, num_keys, keys, *args, count=1):
```

Comment:
We should split out left|right as an argument named direction.
What's the behaviour meant to be when count is 0 (perhaps we shouldn't have a default count)
Can you add type hints for this - I'd like to start using them.